quantization/tests/test_avx2.rs

```diff
@@ -84,4 +84,41 @@ mod tests {
             assert!((score - orginal_score).abs() < error);
         }
     }
+
+    #[test]
+    fn test_l1_avx() {
+        let vectors_count = 129;
+        let vector_dim = 65;
+        let error = vector_dim as f32 * 0.1;
+
+        //let mut rng = rand::thread_rng();
+        let mut rng = rand::rngs::StdRng::seed_from_u64(42);
+        let mut vector_data: Vec<Vec<f32>> = Vec::new();
+        for _ in 0..vectors_count {
+            let vector: Vec<f32> = (0..vector_dim).map(|_| rng.gen_range(-1.0..1.0)).collect();
```

Comment:
```suggestion
            let vector: Vec<f32> = (0..vector_dim).map(|_| rng.gen_range(-1.0..=1.0)).collect();
```

Nitpick: we might want `..=` here to be inclusive.

If so, there are quite a few other places where we want this as well.

Reply:
I had actually changed this because one of me initial attempts of implementation got false positive tests when all the vectors were in [0, 1].

I personally don't think it matters if it includes 1.0 or not in this context, but I could change it of course.